frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts">
+  import { DetailsPage, DetailItem, AdminContent } from '$lib/layout';
+
+  import t, { date } from '$lib/i18n';
+  import { z } from 'zod';
+  import EditableText from '$lib/components/EditableText.svelte';
+  import { Button, type ErrorMessage } from '$lib/forms';
+  import type { PageData } from './$types';
+  import { OrgRole } from '$lib/gql/types';
+  import { useNotifications } from '$lib/notify';
+  import { _changeOrgMemberRole, _changeOrgName, _deleteOrgUser, _deleteOrg, type OrgSearchParams } from './+page';
+  import OrgTabs, { type OrgTabId } from './OrgTabs.svelte';
+  import { getSearchParams, queryParam } from '$lib/util/query-params';
+  import OrgMemberTable from '$lib/components/Orgs/OrgMemberTable.svelte';
+  import { Icon, TrashIcon } from '$lib/icons';
+  import ConfirmDeleteModal from '$lib/components/modals/ConfirmDeleteModal.svelte';
+  import { goto } from '$app/navigation';
+  import { DialogResponse } from '$lib/components/modals';
+
+  export let data: PageData;
+  $: user = data.user;
+  let orgStore = data.org;
+  $: org = $orgStore;
+
+  const queryParams = getSearchParams<OrgSearchParams>({
+    // TODO: Will we want any of the following params that the admin dashboard uses?
+    // userSearch: queryParam.string<string>(''),
+    // showDeletedProjects: queryParam.boolean<boolean>(false),
+    // hideDraftProjects: queryParam.boolean<boolean>(false),
+    // confidential: queryParam.string<Confidentiality | undefined>(undefined),
+    // projectType: queryParam.string<ProjectType | undefined>(undefined),
+    // memberSearch: queryParam.string(undefined),
+    // projectSearch: queryParam.string<string>(''),
+    tab: queryParam.string<OrgTabId>('projects'),
+  });
+  const { queryParamValues } = queryParams;
+
+  $: canManage = user.isAdmin || !!org.members.find(m => m.user?.id === user.id && m.role === OrgRole.Admin)
+
+  const { notifySuccess, notifyWarning } = useNotifications();
+
+  const orgNameValidation = z.string().trim().min(1, $t('org_page.org_name_empty_error'));
+
+  async function updateOrgName(newName: string): Promise<ErrorMessage> {
+    const result = await _changeOrgName({ orgId: org.id, name: newName });
+    if (result.error) {
+      return result.error.message;
+    }
+    notifySuccess($t('org_page.notifications.rename_org', { name: newName }));
+  }
+
+  let deleteOrgModal: ConfirmDeleteModal;
+  async function confirmDeleteOrg(): Promise<void> {
+    const result = await deleteOrgModal.open(org.name, async () => {
+      if (confirm(`Do you really want to delete ${org.name}? There is NO UNDO.`)) { // TODO: en.json
```

Comment:
This is pretty hard core 🤔
If you really think it's a good idea, I think it's critical that you translate these texts.

Reply:
Yeah I think this is overkill as long as we prompt the user to type something in (like DELETE_ORG which we do for projects and such). I also think we can implement a soft delete later on. I'll make an issue for that.

Reply:
Yeah, I do like the idea of a soft delete.

Reply:
Once we move to using an `isDeleted` column on orgs the way we do on projects, the double-confirm won't be necessary. Maybe I'll change it to only trigger if the change would affect more than 10 projects or more than 25 users, that way deleting test orgs that we create during development won't be tedious. And eventually it can go away.

But yes, certainly, it needs to be translated, which is why I put a TODO in there to remind myself of that. (Also why I'm increasingly feeling like it's long past time to put https://github.com/sillsdev/languageforge-lexbox/issues/602 into the next milestone we do, to make sure we don't populate en.json and neglect the translations).

Reply:
Changed my mind and decided that our existing confirm dialog is good enough; by the time we have orgs with many projects in them, we'll likely have a soft-delete option. So I removed this in commit 909070b5.